bootstrap.sh

```diff
@@ -1,20 +1,21 @@
 #!/bin/bash
-kubectl apply -f .infrastructure/mysql/ns.yml
-kubectl apply -f .infrastructure/mysql/configMap.yml
-kubectl apply -f .infrastructure/mysql/secret.yml
-kubectl apply -f .infrastructure/mysql/service.yml
-kubectl apply -f .infrastructure/mysql/statefulSet.yml
+# kubectl apply -f .infrastructure/mysql/ns.yml
+# kubectl apply -f .infrastructure/mysql/configMap.yml
+# kubectl apply -f .infrastructure/mysql/secret.yml
+# kubectl apply -f .infrastructure/mysql/service.yml
+# kubectl apply -f .infrastructure/mysql/statefulSet.yml
 
-kubectl apply -f .infrastructure/app/ns.yml
+# kubectl apply -f .infrastructure/app/ns.yml
```

Comment:
The namespace for the app is commented out. Make sure the namespace already exists or is created by another means before applying other resources.